packages/eventual-send/src/local.js

```diff
@@ -0,0 +1,39 @@
+const q = JSON.stringify;
```

Comment:
You use `q` to quote, among other things, the `method` name. But
```js
JSON.stringify(Symbol.asyncIterator) // undefined
```
Perhaps something like
```suggestion
const q = v => JSON.stringify(String(v));
```
except that won't work in the array case below. Again JSON.stringify isn't quite enough. But I also understand why you're reluctant to depend on `assert`. I don't have a good suggestion, but this does need to work with symbol-named properties. We will shortly allow many more of these besides `Symbol.asyncIterator`.

Reply:
I use `assert` elsewhere now, so maybe it is time to use it here, too.